README.md

```diff
@@ -97,57 +104,16 @@ You can also specify explicitly what services are included in the ear by passing
         
 ######Altering targeted application server
-For some application server deployments the generated .ear needs different dependencies. This readme describes building and deploying on GlassFish. See the CONNECT wiki for details on other application servers.
-
-###Setup Glassfish, MySQL & Deploy CONNECT
-These steps will install and configure a Glassfish Application Server, prepare your MySQL databases and deploy CONNECT for use. Lets get started.
-
-Navigate to the <CONNECT_CLONE_DIR>/Product/Install directory
-
-        $ cd <CONNECT_CLONE_DIR>/Product/Install
-
-Next, copy install.properties to local.install.properties and update with your local information. Generally this just specifying where
-you want GlassFish to install.
-
-        $ cp install.properties local.install.properties
-
-
-####Setup MySQL Databases
-These steps will install and configure a Glassfish Application Server to deploy and use CONNECT. Lets get started.
-
-Navigate to the <CONNECT_CLONE_DIR>/Product/Install directory
-
-        $ cd <CONNECT_CLONE_DIR>/Product/Install
-
-Next, copy install.properties to local.install.properties and update with your local information. Generally this is just specifying a password for mysql.
-
-        $ cp install.properties local.install.properties
+For some application server deployments the generated .ear needs different dependencies. Follow below wiki links for CONNECT supported application server build and deployment steps.
 
-Lastly, we're going to install the MySQL databases needed for CONNECT.
-
-        $ ant install.databases
-
-The "mysql-connector-java-5.1.10.jar" must be copied into the following folder, for container managed database resources
-
-        <GLASSFISH_HOME>/domains/domain1/lib/ext
-
-####Setup GlassFish
-These steps will install and configure a Glassfish Application Server to deploy and use CONNECT. Lets get started.
-
-Navigate to the <CONNECT_CLONE_DIR>/Product/Install directory
-
-        $ cd <CONNECT_CLONE_DIR>/Product/Install
-
-Next, copy install.properties to local.install.properties and update with your local information. Generally this just specifying where
-you want GlassFish to install.
-
-        $ cp install.properties local.install.properties
-
-Lastly, we're going to install the Glassfish Application Server
-
-        $ ant install
+  * Setup GlassFish, MySQL & Deploy CONNECT:[Glassfish Install](https://connectopensource.atlassian.net/wiki/display/CONNECT4/Building+CONNECT+4.4+from+Source) 
+  * WebSphere, MySQL & Deploy CONNECT:[WebSphere Setup](https://connectopensource.atlassian.net/wiki/pages/viewpage.action?pageId=17203313)
+  * JBoss7, MySQL & Deploy CONNECT:[JBoss7 Setup](https://connectopensource.atlassian.net/wiki/pages/viewpage.action?pageId=26214665)
+  * WebLogic, MySQL & Deploy CONNECT:[WebLogic Setup](https://connectopensource.atlassian.net/wiki/pages/viewpage.action?pageId=17203304)
```

Comment:
Suggestion: "MySQL" and "Deploy CONNECT" are implied by the sentence above this list, so you could just use:

- GlassFish: <link>
- WebSphere: <link>

etc.